### DISCUSSION TITLE
Treshold line in combined chart

### DISCUSSION THREAD

**Discussion by ivuceticll** [patrykandpatrick/vico#291] [Questions]
Please can you help me, how I can set threshold line in combined chart. I have two column chart and I want to show threshold line, or is it possible to change guideline just on one line ?

**Comment by patrickmichalik** [comment]
Hello! Just like `ColumnChart` and `LineChart`, `ComposedChart` extends `BaseChart`. Thus, it has the `addDecoration`, `setDecorations`, and `removeDecorations` functions.
